### PR TITLE
Added `image_augmentation_torch.ipynb` for `image_augmentation_jax.ipynb`

### PR DESCRIPTION
**Pull request by susnato**
## Description

Converted `image_augmentation_torch.ipynb` to `image_augmentation_jax.ipynb`

### Colab link
https://colab.research.google.com/drive/19aCO4g8K5CPGFYOOOaKHXNhtbt--lYVL?usp=sharing

### Issue 

https://github.com/probml/pyprobml/issues/686

## Checklist: 

- [✓] Performed a self-review of the code

- [✓]  Tested on Google Colab.

### CODE REVIEW

**Comment by susnato** [comment]
Can you please review my code? @murphyk @mjsML @gerdm

**Comment by susnato** [comment]
Sorry, as you can see above I created some mess with commits because I forgot to make another branch. Well, it's fine now and ready.

**Comment by murphyk** [comment]
This looks good. However, a few things could be improved:
- use https://github.com/probml/probml-notebooks/blob/main/images/cat_dog.jpg instead of `https://www.dropbox.com/s/7pzstlslgbatgvu/cat_dog_sample.zip?dl=0 -q`, in case your dropbox account expires :) 
- It would be nice to eliminate all dependence on the d2l library. It seems to only be used for trivial things like `d2l.show_images`. Maybe you can copy or rewrite that function?
- It might also be nice to replace the torch CIFAR10 dataloader with TFDS.

**Comment by susnato** [comment]
Hi, @murphyk I made changes as you asked. The Colab link is the same (https://colab.research.google.com/drive/19aCO4g8K5CPGFYOOOaKHXNhtbt--lYVL?usp=sharing), could you please review it and tell if everything is alright or not?

**Comment by murphyk** [comment]
Thanks, this is much better. However the output of the `show_images` function is a bit ugly (see below).
I think you can use https://scikit-image.org/docs/stable/api/skimage.util.html#skimage.util.montage to  make a grid out of the individual images, so that there is less white space between them (and no axis lines). 
Screen Shot 2022-03-24 at 10 40 23 AM

**Comment by susnato** [comment]
Hi, @murphyk I used the `montage` function in image grids instead of `show_images`, the Colab link is the same (https://colab.research.google.com/drive/19aCO4g8K5CPGFYOOOaKHXNhtbt--lYVL?usp=sharing) please review it and tell if anything needs to be changed or not?

This is the new output : 
Screenshot 2022-03-28 at 10-01-59 Google Colaboratory
